src/vivarium_gates_child_iv_iron/data/loader.py

```diff
@@ -291,8 +324,9 @@ def load_remission_rate_from_duration(key: str, location: str) -> pd.DataFrame:
         }[key]
     except KeyError:
         raise ValueError(f'Unrecognized key {key}')
+    step_size = .5 / 365 # years
     duration = get_data(cause.DURATION, location)
-    remission_rate = 1 / duration
+    remission_rate = (-1/step_size)*np.log(1-step_size/duration)
```

Comment:
spacing here is bad. Maybe just run isort [file] && black [file] ?

Reply:
I have run both and fixed formatting.